packages/core/admin/admin/src/pages/UsecasePage/index.js

```diff
@@ -0,0 +1,148 @@
+import { Main } from '@strapi/design-system/Main';
+import { Flex } from '@strapi/design-system/Flex';
+import { Box } from '@strapi/design-system/Box';
+import { Stack } from '@strapi/design-system/Stack';
+import { Typography } from '@strapi/design-system/Typography';
+import { Select, Option } from '@strapi/design-system/Select';
+import { TextInput } from '@strapi/design-system/TextInput';
+import { TextButton } from '@strapi/design-system/TextButton';
+import { Button } from '@strapi/design-system/Button';
+import Logo from '../../components/UnauthenticatedLogo';
+import UnauthenticatedLayout, { LayoutContent } from '../../layouts/UnauthenticatedLayout';
+
+export const options = [
+  { label: 'Front-end developer', value: 'front_end_developer' },
+  { label: 'Back-end developer', value: 'back_end_developer' },
+  { label: 'Full-stack developer', value: 'full_stack_developer' },
+  { label: 'Content Manager', value: 'content_manager' },
+  { label: 'Content Creator', value: 'content_creator' },
+  { label: 'Other', value: 'other' },
+];
+
+const TypographyCenter = styled(Typography)`
+  text-align: center;
+`;
+
+const UsecasePage = () => {
+  const toggleNotification = useNotification();
+  const { push, location } = useHistory();
+  const { formatMessage } = useIntl();
+  const [persona, setPersona] = useState();
+  const [personaOther, setPersonaOther] = useState('');
+
+  const isOther = persona === 'other';
+  const isComingFromRegister = location.state?.fromRegister;
+
+  useEffect(() => {
+    if (!isComingFromRegister) {
+      push('/');
+    }
+  }, [isComingFromRegister, push]);
+
+  const handleSubmit = skipPersona => {
+    try {
+      const {
+        state: { email, firstAdmin, firstname },
+      } = location;
+
+      axios({
+        method: 'POST',
+        url: 'https://analytics.strapi.io/register',
+        data: {
+          email,
+          username: firstname,
+          firstAdmin,
+          persona: skipPersona ? undefined : persona,
+          personaOther: skipPersona ? undefined : personaOther,
+        },
+      });
+
+      toggleNotification({
+        type: 'success',
+        message: {
+          id: 'Usecase.notification.success.project-created',
+          defaultMessage: 'Project has been successfully created',
+        },
+      });
+      push('/');
+    } catch (err) {
+      // Silent
+    }
+  };
+
+  if (!isComingFromRegister) {
+    return null;
+  }
+
+  return (
+    <UnauthenticatedLayout>
+      <Main>
+        <LayoutContent>
+          <form onSubmit={() => handleSubmit(false)}>
+            <Flex direction="column" paddingBottom={7}>
+              <Logo />
+              <Box paddingTop={6} paddingBottom={1} width={pxToRem(250)}>
+                <TypographyCenter variant="alpha" as="h1">
+                  {formatMessage({
+                    id: 'Usecase.title',
+                    defaultMessage: 'Tell us a bit more about yourself?',
```

Comment:
I think this shouldn't be a questions. If so, I'd formulate it as a questions, e.g.

```suggestion
                    defaultMessage: 'Could you tell us a bit about yourself?',
```

Reply:
I went for that at first, but it takes way too much space as an H1 (or H3 can't remember) in this small space 🤯